content/en/data_jobs/databricks.md

```diff
@@ -27,17 +28,60 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
+1. In the **Select products to set up integration** section, make sure the Data Jobs Monitoring product is **Enabled**.
+1. In the **Datadog Agent Setup** section, chooose to either
+  - [Have Datadog install and manage a global init script](?tab=datadogmanagedglobalinitscriptrecommended#install-the-datadog-agent) in the workspace (recommended).
+  - Or, [manaully install and manage a global init script](?tab=manuallyinstallaglobalinitscript#install-the-datadog-agent).
+  - Or, [manaully install and manage an init script on a specific cluster](?tab=manuallyinstallonaspecificcluster#install-the-datadog-agent).
 
+### Install the Datadog Agent
 
-### Install the Datadog Agent on your Databricks cluster(s)
+The Datadog Agent must be installed on Databricks clusters to monitor Databricks jobs that run on the clusters.
 
-You can choose to install the Agent globally, or on a specific Databricks cluster.
+<div class="alert alert-warning">The init script installs the agent when clusters start. All-purpose clusters must be restarted for the init script to install the Datadog Agent.</div>
 
 {{< tabs >}}
-{{% tab "Global init (Recommended)" %}}
+{{% tab "Datadog managed global init script (Recommended)" %}}
+
+Datadog can install and manage a global init script in the Databricks workspace. The Datadog Agent is installed on all clusters in the workspace, when they start.
+
+#### When adding a new Databricks workspace
+
+1. In the **Select products to set up integration** section, make sure the Data Jobs Monitoring product is **Enabled**.
+1. In the **Datadog Agent Setup** section, select the **Managed by Datadog** toggle button.
+1. Click **Select API Key** to either select an existing Datadog API key or create a new Datadog API key.
+   {{< img src="data_jobs/databricks/configure-data-jobs-monitoring-new.png" alt="In the Datadog-Databricks integration tile, Datadog Agent Setup when adding a Databricks workspace. Datadog can install and manage a global init script." style="width:100%;" >}}
+
+#### To modify a Databricks workspace account in the Datadog-Databricks integration
+
+1. On the **Configure** tab, click the workspace in the list of workspaces
+1. Click the **Configured Products** tab
+1. Make sure the Data Jobs Monitoring product is **Enabled**.
+1. In the **Datadog Agent Setup** section, select the **Managed by Datadog** toggle button.
+1. Click **Select API Key** to either select an existing Datadog API key or create a new Datadog API key.
+   {{< img src="data_jobs/databricks/configure-data-jobs-monitoring-existing.png" alt="In the Datadog-Databricks integration tile, Datadog Agent Setup for a Databricks workspace already added to the integration. Datadog can install and manage a global init script." style="width:100%;" >}}
+
+{{% /tab %}}
+
+{{% tab "Manually install a global init script" %}}
+
+
+When adding a new Databricks workspace:
```

Comment:
@tiffon remove all of this new next section talking about the integration tile first as part of the manual instructions. I see the logic of why you put it here for completeness assuming the user is starting at the tile, but everything they need to actually do to complete this is in the Databricks console, so we should just tell them to go there from the start and simplify this. I would be annoyed as a user following these steps, and going to the tile, only to not have anything to do/change there.